src/controllers/mainController.ts

```diff
@@ -232,6 +232,7 @@ export default class MainController implements vscode.Disposable {
                 this._context.subscriptions.push(
                     vscode.commands.registerCommand(
                         Constants.cmdConnectObjectExplorerNode, async (node: ConnectTreeNode) => {
+                        self._objectExplorerProvider.currentNode = node.parentNode;
```

Comment:
Just so we know, any chance a node won't have a parent?

Reply:
not for a `ConnectTreeNode`